src/basic_recipes/title.jl

```diff
@@ -0,0 +1,30 @@
+@recipe(Title, titletext, plot) do scene
+    Theme(
+        textsize=40,
+        raw=true,
+        font=theme(scene, :font),
+        align = (:center, :bottom), 
+    )
+end
+
+function convert_arguments(::Type{<:Title}, titletext::AbstractString, plot::Scene)
```

Comment:
This doesn't seem to do much, also you don't have to return the titletext _and_ the plot.  
Something like:
```julia
convert_arguments(::Type{<:Title}, titletext::AbstractString) = string(titletext)
``` 
seems to be what you're aiming for here - but please correct me if I'm wrong!

Reply:
It is not clear to me, when precisely `convert_arguments` is called and what is the expected output. I think I stole returning the plot as well from an example somewhere... Anyway I deleted the `convert_arguments` function.

Reply:
`convert_arguments` is basically used when you want to define a conversion from one Type to another, before it gets into the plotting function - say you wanted to, for a specific plot type, convert all numbers to strings, but not need the user to bother with that, you could do it internally using convert_arguments.  It's basically a way to reduce redundancy.

Reply:
Ok thanks!